.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "bracketSpacing": true,
+  "printWidth": 120,
+}
```

Comment:
Did you mean to commit this? Looks like an accident. (This appears to be a user-specific preference file. Probably shouldn't be committed.)

Reply:
This is for the README.md so that any future changes to it follow the same style. I don't care what the style is, I just care that it's documented.

Reply:
```bash
prettier --write README.md
```

Reply:
What tools support this file though? It's the first I've heard of it...

Reply:
https://prettier.io

You don't worry about this much in the Go world because you've got `go fmt`, but `prettier` is an all-in-one for docs, js, python, html, css, etc, etc.